.github/workflows/comment-on-pr.yml

```diff
@@ -0,0 +1,58 @@
+name: Comment on PR
+
+on:
+  workflow_call:
+    secrets:
+      gh_token:
+        required: true
+
+jobs:
+    comment-on-pr:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Get Artifact URL & PR Info
+            env:
+              GITHUB_TOKEN: ${{ secrets.gh_token }}
+              OWNER: ${{ github.repository_owner }}
+              REPO: ${{ github.event.repository.name }}
+              WORKFLOW_RUN_EVENT_OBJ: ${{ toJSON(github.event.workflow_run) }}
+            run: |
+              PREVIOUS_JOB_ID=$(jq -r '.id' <<< "$WORKFLOW_RUN_EVENT_OBJ")
+              echo "Previous Job ID: $PREVIOUS_JOB_ID"
+              echo "PREVIOUS_JOB_ID=$PREVIOUS_JOB_ID" >> "$GITHUB_ENV"
+    
+              PR_NUMBER=$(jq -r '.pull_requests[0].number' \
+                  <<< "$WORKFLOW_RUN_EVENT_OBJ")
```

Comment:
wahou, first time I see so many usages of `<<<` , is it really needed ?

from `man bash`:

```
Here Strings
       A variant of here documents, the format is:

              [n]<<<word

       The word undergoes tilde expansion, parameter and variable expansion, command substitution, arithmetic expansion, and quote removal.  Pathname expansion and word splitting are not performed.  The result is supplied as a
       single string, with a newline appended, to the command on its standard input (or file descriptor n if n is specified).
```

Reply:
I used this snippet of code https://github.com/orgs/community/discussions/51403#discussioncomment-5515349 which used `<<<`. I am unsure if it is absolutely necessary